stock_quant_view_reservation/views/stock_quant.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="stock_quant_tree_view">
+        <field name="name">stock.quant.tree (in stock_quant_view_reservations)</field>
+        <field name="model">stock.quant</field>
+        <field name="inherit_id" ref="stock.view_stock_quant_tree_editable" />
```

Comment:
I think this should be view_stock_quant_tree and not view_stock_quant_tree_editable. 

At least I don't get to see the eye icon if it's inherit from view_stock_quant_tree_editable